docs/installation.md

```diff
@@ -146,6 +146,7 @@ Then, in the `pyserini` clone, use `pip` to add an ["editable" installation](htt
 
 ```bash
 pip install -e .
+# If an error occurred during installation, try this: python -m pip install -e .
```

Comment:
I think this is idiosyncratic to your setup... you're the first to have encountered the error you relayed to me over email...